test/trails_SUITE.erl

```diff
@@ -1,7 +1,5 @@
 -module(trails_SUITE).
 
--author('elbrujohalcon@inaka.net').
-
```

Comment:
Elvis, right? Lemme know if you prefer this to be present elsewhere, e.g. in a comment.

Reply:
Kill it with fire!